File: main/src/main/java/com/badlogic/test/MainActivity.java
package com.badlogic.test;

import android.app.Activity;
import android.os.Bundle;
import android.view.Window;

import com.badlogic.utils.ALog;
import com.badlogic.utils.Tools;


public class MainActivity extends Activity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        ALog.i("wjw","--->"+ Tools.getVersion(this));
        ///005e
    }


}

File: main/src/main/java/com/teaphy/diffutildemo/MainActivity.kt
package com.teaphy.diffutildemo
//com.teaphy.diffutildemo.MainActivity
import android.app.Activity
import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.badlogic.ui.R
import kotlinx.android.synthetic.main.differ_activity_main.*
import org.jetbrains.anko.startActivity


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.differ_activity_main)

        initData()

        initView()

        setListener()

    }

    private fun initData() {

    }

    private fun initView() {


    }

    private fun setListener() {
        btnDiff.setOnClickListener {
            startActivity<DiffActivity>()
        }

        btnAld.setOnClickListener {
            startActivity<AsyncListDifferActivity>()
        }

        btnLa.setOnClickListener {
            startActivity<ListAdapterActivity>()
        }

        btnSl.setOnClickListener {
            startActivity<SortedListActivity>()
        }
    }



}
